corehq/blobs/migrate.py

```diff
@@ -199,19 +202,34 @@ def __init__(self, slug, doc_types, migrate_func):
         self.migrate_func = migrate_func
 
     def migrate(self, filename=None):
-        return migrate(self.slug, self.doc_types, self.migrate_func, filename)
+        return migrate(self.slug, self.doc_type_map, self.migrate_func, filename)
+
+    @property
+    def doc_type_map(self):
+        def itemize(item):
+            """get `(doc_type, model_class)` pair"""
+            if isinstance(item, tuple):
+                return item
+            return item.__name__, item
+        return dict(itemize(item) for item in self.doc_types)
 
 
 MIGRATIONS = {m.slug: m for m in [
     Migrator("saved_exports", [SavedBasicExport], migrate_from_couch_to_blobdb),
     Migrator("migrate_backend", [SavedBasicExport], migrate_blob_db_backend),
+    Migrator("applications", [
+        Application,
```

Comment:
@millerdev I just checked and you're right that apps that are copies of another app (i.e. app builds) don't get their version incremented, so I agree it's fine to save.